assignment/README.md

```diff
@@ -12,12 +12,18 @@ Applications are ultimately just tools. What problem or question does
 your application attempt to resolve or grapple with? How does your
 application speak to this problem/question?
 
+**The city of Louisville is attempting to mitigate its structural fires. @mayutanaka (Mayu) and I have been asked by the Fire Department Chief to develop an algorithm that will help to predict where these future fires may occur. The application will show a map of our predicted risks, as well as locations at which the city can allocate educational outreach. Community access points, such as schools, churches, and community centers, will be the locations for the educational outreach.**
+
 ## The data
 
 Geospatial applications are all about working with data. What datasets
 would you plan/like to use? If the data you'll be working with isn't
 already stored in a way that you can use, how will you be storing your data?
 
+**We'll be using datasets that we have compiled ourselves, though they are sourced from multiple locations: property valuation data and open data. Our dataset merges these data sources with our predictive data.**
+
+**As for storage, we were going to attempt to use Github to do so, as that's what we've used in previous class examples. However, if this proves to be insufficient, we will need to look at alternatives.**
```

Comment:
GH will *probably* be sufficient to your needs. If it gives you any trouble, let me know